docs/INSTALLING.md

```diff
@@ -96,6 +96,10 @@ Point your browser (or make a direct request) to `http://api.localhost:49235/use
 }
 ```
 
+If your browser does not support localhost loopback (address not found error), please use `http://api.lvh.me:49235/users` address instead (lvh.me resolves itself and all subdomains to your `localhost` accordingly).
```

Comment:
I would actually just change L88 to say `http://api.lvh.me:49235/users` with the note that:

> lvh.me resolves itself and all subdomains to your `localhost` accordingly

If someone more advanced wants to use `api.localhost`, then they can without probably much need for fruther explanation.